### ISSUE TITLE
Inconsistent physical boxes and immovable trash bags

### ISSUE THREAD

**Issue by without-ordinary** [Archival1/CV-BugReporting#17]
**Describe the bug**
Most of the cardboard boxes are movable, including some that are near the player's height. There are some mid-sized rectangular boxes that are an not physical.

The small trash bag props are an immovable force.

**To Reproduce**
1. Load Station.
2. Attempt to move the the mid sized boxes near spawn.

1. Load Office
2. Go to recombob room.
3. Try to move boxes near doorway to hall and note how they will get stuck on the small trash bag.

**Expected behavior**
Floor clutter props smaller than the small cardboard box to be movable.
Boxes to have some consistency in movability, like most outside of the large ones in warehouse maze. Possibly a second, visually different, prop model for immovable boxes for better hinting.

**Screenshots**
![20240122011452_1](https://github.com/Archival1/CV-BugReporting/assets/20104884/790ccf1f-1ce4-4620-843c-475cb105a008)
![20240122011649_1b](https://github.com/Archival1/CV-BugReporting/assets/20104884/10e0480c-f900-4315-9902-d1cc6af459e5)

**Desktop (please complete the following information):**
 - OS: Linux
 - Graphics Card: NVIDIA 3090
 - Version: 1/19/2024_cef91e5c

**Comment by naelstrof** [comment]
Agreed, we should've made some custom textures at least to indicate static boxes. Many of our levels majorly use the box prop as it was the only prop we had for a very long time! Haha, nobody wanted to do prop work.

**Comment by without-ordinary** [comment]
> Haha, nobody wanted to do prop work.
I hear that a lot actually.

I should really go back to learning Blender again, recreating static props/mechanical components in CAD was something I was doing for years.
